opendebates/templates/flatpages/social_override.html

```diff
@@ -0,0 +1,22 @@
+{% extends "opendebates/base.html" %}
+
+{% block metadata %}
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:title" content="HAPPENING NOW! Watch history be made.">
+<meta name="twitter:description" content="Well, this is awesome. I'm watching Democratic Congressman Alan Grayson debate Republican Congressman David Jolly in Florida. It's the first time ever U.S. Senate candidates are participating in an Open Debate where all questions where submitted and voted on by the public -- and moderators must ask top-voted questions. Watch it with me...">
```

Comment:
Hah.  Yes it should.  (In the FB one too, obviously.)  Thanks for catching
that...

On Mon, Apr 25, 2016 at 11:41 AM, Vinod Kurup <notifications@github.com>
wrote:

> In opendebates/templates/flatpages/social_override.html
> <https://github.com/caktus/django-opendebates/pull/171#discussion_r60935638>
> :
>
> > @@ -0,0 +1,22 @@
> > +{% extends "opendebates/base.html" %}
> > +
> > +{% block metadata %}
> > +<meta name="twitter:card" content="summary_large_image">
> > +<meta name="twitter:title" content="HAPPENING NOW! Watch history be made.">
> > +<meta name="twitter:description" content="Well, this is awesome. I'm watching Democratic Congressman Alan Grayson debate Republican Congressman David Jolly in Florida. It's the first time ever U.S. Senate candidates are participating in an Open Debate where all questions where submitted and voted on by the public -- and moderators must ask top-voted questions. Watch it with me...">
>
> I think the second 'where' should be 'were'? Otherwise this looks fine.
> @ejucovy <https://github.com/ejucovy>, I'm happy to make the change and
> start the testing deploy. Just want your confirmation.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/caktus/django-opendebates/pull/171/files/6a4095ce798bbcd82876f920c2b73e8d82a95c21#r60935638>
>